api/admin/controller/__init__.py

```diff
@@ -1243,7 +1243,6 @@ def circulation_events(self):
         events = map(lambda result: {
             "id": result.id,
             "type": result.type,
-            "patron_id": result.foreign_patron_id,
```

Comment:
Because I removed the `foreign_patron_id` field from the `patrons` table, we need to stop gathering this information and sending it out to the controllers. It looks like this information was never actually used in the display (which makes sense since we never really had it), but I'd like you to verify this.

Reply:
Yes, `patron_id` is not displayed in the web admin. We would have to make a minor change to the front-end since it expects `patron_id` in this object (not a big deal but just making a note).